pom.xml

```diff
@@ -51,10 +51,13 @@
     </distributionManagement>
 
     <properties>
-        <!-- Build will not fail if error occurs during javadoc generation -->
-        <!--<maven.javadoc.failOnError>false</maven.javadoc.failOnError>-->
         <!-- Build will skip the javadoc generation -->
-        <maven.javadoc.skip>true</maven.javadoc.skip>
+        <!--<maven.javadoc.skip>true</maven.javadoc.skip>-->
```

Comment:
can delete these if not needed

Reply:
will check if validation rules passes this time in Maven Central release task. After that, these can be cleared.